blog/content/post/2021-02-05-testing.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Manual vs Automated Testing
+abstract: 'Pros and cons (if there are any) of automated testing and the process of transparent validation.'
+cover: /covers/test-cover.png
+author:
+  - piotr.konkol
+  - kamila.banecka
+layout: post
+published: true
+date: 2021-02-05
+archives: "2021"
+
+tags:
+  - testing
+  - firmware
+categories:
+  - Firmware
+  - IoT
+  - Miscellaneous
+  - OS Dev
+  - App Dev
+  - Security
+  - Manufacturing
+
+---
+
+#### The Importance of test automation
+
+
+ <div style="text-align: justify"> It is common understanding that software, or – in our case – firmware must be
```

Comment:
Can't we instruct hugo to justify the paragraphs? @artur-rs ?

Reply:
@miczyg1 we can add simple css change:
```
.post-content p {
    text-align: justify;
}
```
but do we want that in all posts? @KBanecka ?

Reply:
@KBanecka if you want to test that, find `bulma.css` file and add or replace above method in the code, then do the local preview of the blog post

Reply:
dodałam zmianę